packages/trpc/server/routers/viewer/organizations/adminUpdate.handler.ts

```diff
@@ -49,13 +50,33 @@ export const adminUpdateHandler = async ({ input }: AdminUpdateOptions) => {
     data.metadata = mergeMetadata({
       // If we save slug, we don't need the requestedSlug anymore
       requestedSlug: undefined,
-      ...input.metadata,
     });
   }
 
-  const updatedOrganisation = await prisma.team.update({
-    where: { id },
-    data,
+  const updatedOrganisation = await prisma.$transaction(async (tx) => {
+    const updatedOrganisation = await tx.team.update({
+      where: { id },
+      data,
+    });
+
+    if (organizationSettings || existingOrg.organizationSettings) {
+      await tx.organizationSettings.update({
+        where: {
+          organizationId: updatedOrganisation.id,
+        },
+        data: {
+          isOrganizationConfigured:
+            organizationSettings?.isOrganizationConfigured ||
+            existingOrg.organizationSettings?.isOrganizationConfigured,
+          isOrganizationVerified:
+            organizationSettings?.isOrganizationVerified ||
+            existingOrg.organizationSettings?.isOrganizationVerified,
+          orgAutoAcceptEmail:
+            organizationSettings?.orgAutoAcceptEmail || existingOrg.organizationSettings?.orgAutoAcceptEmail,
```

Comment:
This might not be what's intended. It's possible that organizationSettings is set but not all props are in it are set. So, the result could be different. The intention is to fallback for each and every property of organizationSettings individually